### DISCUSSION TITLE
Tick marks  on valuator.

### DISCUSSION THREAD

**Discussion by rfoxkendo** [fltk-rs/fltk-rs#1289] [General]
Fltk's native valuators have a draw_tick method which specifies intervals in the valuator at which tick marks are drawn.  I could not find this functionality in the docs... is it exposed somewhere?  Are there plans for it or should I submit a feature request?
Thanks,
Ron.

**Comment by MoAlyousef** [comment]
Hi
I think the draw_ticks method was part of fltk2 (from a cursory search online), the FLTK 1.4 source code (which this crate binds to) doesn’t have it.
